.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8']
+
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@master
+      - uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool }}
+
+      - name: Run regression testing
+        run: tarantool ./test.lua
+
+      - name: Setup LuaJIT
+        run: sudo apt install -y luajit
+
+      - name: Load checks module in LuaJIT
+        run: luajit -l checks
```

Comment:
I guess this part will be called for each version of Tarantool. But it's enough to do it only once.

Reply:
Command takes about 2ms for execution, but ok I'll move it to a separate job.